lib/beaker/host/unix/pkg.rb

```diff
@@ -17,28 +17,30 @@ def check_for_command(name)
     end
   end
 
-  def check_for_package(name)
+  def check_for_package(name, opts = {})
+    opts = {:accept_all_exit_codes => true}.merge(opts)
+    opts[:prepend_cmds] ? pc = "#{opts[:prepend_cmds]} " : pc = ""
```

Comment:
I think that we should create command object support for this option.  Look in command.rb and see how it handles adding 'cmd.exe /c' to some commands - :prepend_cmds could be passed through and handled there and then it will be supported across the board and not just in this corner of beaker.